lib/ddtrace/contrib/rails/action_view.rb

```diff
@@ -60,29 +82,6 @@ def self.start_render_partial(*)
           Datadog::Tracer.log.error(e.message)
         end
 
-        def self.render_template(_name, start, finish, _id, payload)
-          key = get_key('render_template')
-          return unless Thread.current[key]
-          Thread.current[key] = false
-
-          # finish the tracing and update the execution time
-          tracer = ::Rails.configuration.datadog_trace.fetch(:tracer)
-          span = tracer.active_span()
```

Comment:
We don't rely anymore in the opening/closing order. This ensures that our spans are properly closed in the right way whatever happens.